sample-new-architecture/ios/sampleNewArchitecture.xcodeproj/project.pbxproj

```diff
@@ -523,6 +523,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = sampleNewArchitecture;
 				RCT_NEW_ARCH_ENABLED = 1;
+				SENTRY_PROFILING_ENABLED = 1;
```

Comment:
Will that be required in People's apps?

Reply:
This was just from testing. I've removed it.
People will need to set `SENTRY_PROFILING_ENABLED=1` when installing pods. This will add compiler flag and the profiling code will be included in the build.

Reply:
I see, wondering if there's a way to avoid this, let's pair up.